sqlglot/dialects/bigquery.py

```diff
@@ -419,6 +419,7 @@ class Generator(generator.Generator):
         RENAME_TABLE_WITH_DB = False
         NVL2_SUPPORTED = False
         UNNEST_WITH_ORDINALITY = False
+        LOG_BASE_FIRST = False
```

Comment:
Since this flag is used in both the `Parser` and the `Generator`, can we make it a `Dialect` constant instead? Check for example how `ALIAS_POST_TABLESAMPLE` is handled.

Reply:
Lemme give it shot.

Reply:
Done!

Reply:
Added for consistency: https://github.com/tobymao/sqlglot/commit/79fc4d1a990cd4b86d55e5a1ce8c37ce5556765c